compiler/damlc/tests/BUILD.bazel

```
@@ -335,6 +335,8 @@ da_haskell_test(
         "//compiler/damlc/pkg-db",
         "//compiler/damlc/stable-packages",
         "//compiler/scenario-service/server:scenario_service_jar",
+        "//daml-script/daml:daml-script-1.15.dar",
```

since 1.15 is the default LF version, this should also work while being a bit more future-proof

```suggestion
        "//daml-script/daml:daml-script.dar",
```

---

ah, I noticed that would break `withDamlScriptDep`, since it adds the LF version explicitly. Perhaps `withDamlScriptDep` should take a `Maybe LF.Version` instead? not terribly urgent, though

---

Yeah that was the reason for the 115, but I hadn't considered the def Lf version changing, I'll fix this up

---

thanks, it's fine if you do it in a separate PR :)